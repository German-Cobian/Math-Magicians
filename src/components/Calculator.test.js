import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Calculator from './Calculator';

describe('<Calculator />', () => {
  test('it should match snapshot', () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
