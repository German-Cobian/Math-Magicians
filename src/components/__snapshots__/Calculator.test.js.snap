// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calculator /> it should match snapshot 1`] = `
<div
  className="calculator-body"
>
  <div
    className="screen"
  >
    0
  </div>
  <div
    className="keyboard"
  >
    <div
      className="row"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="row last-row"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
